spec/unit/configurer_spec.rb

```diff
@@ -359,6 +359,13 @@
       @agent.apply_catalog(@catalog, {:report => report})
     end
 
+    it "should record a total run time" do
+      report = Puppet::Transaction::Report.new
+      @catalog.stubs(:apply).with(:report => report)
+      report.expects(:add_times).with(:total, anything())
```

Comment:
I was able to get this test to react to adding/removing the total time entry here.